app/addons/documents/shared-resources.js

```diff
@@ -27,7 +27,7 @@ define([
       return FauxtonAPI.constants.DOC_URLS.GENERAL;
     },
     url: function (context) {
-      if (context === undefined) {
+      if (!context) {
```

Comment:
Again, slight logic change here (if context is 0 or false this will execute) This okay?

I kinda like _.isUndefined(). Verbose but explicit!